pkg/postage/batchstore/store.go

```diff
@@ -60,6 +60,15 @@ func New(st storage.StateStorer, unreserveFunc unreserveFn) (postage.Storer, err
 	return &store{st, cs, rs, unreserveFunc}, nil
 }
 
+func (s *store) GetReserveState() *postage.Reservestate {
+	return &postage.Reservestate{
+		Radius:    s.rs.Radius,
+		Available: s.rs.Available,
+		Outer:     *s.rs.Outer,
```

Comment:
`big.Int`s are meant to be passed as pointers, not as values, so it is needed to create a new `big.Int` and give it the value which is currently in the existing fields.
Also, there is a typo in the line below, where `Outer` is dereferenced as value for Inner